doc/integrator/extend_application.rst

```diff
@@ -50,6 +63,29 @@ Will be filled later.
 Custom JavaScript file
 ----------------------
 
+In this example we will add a button in the tools bar, that open tool panel, that can be used to send a feedback.
+
+The tool button should be an instance of
+`gmfapi.elements.ToolButtonElement<https://camptocamp.github.io/ngeo/|main_branch|/apidoc/classes/srcapi_elements_ToolButtonElement.default.html>`_.
+
+In the example he will directly use tool button will be an instance of
```

Comment:
I don't know what you mean here ... you merged two sentences ? :-)

Reply:
the => this ...